src/pkg/bundle/deploy.go

```diff
@@ -34,6 +36,7 @@ type PkgOverrideMap map[string]map[string]map[string]interface{}
 
 // templatedVarRegex is the regex for templated variables
 var templatedVarRegex = regexp.MustCompile(`\${([^}]+)}`)
+var includedSources = make(map[types.ValueSources]types.ValueSources, 0)
```

Comment:
I think you can remove the `0` arg here + include a comment for this global var

Reply:
On second thought, can we make `includedSources` part of the DeployOpts?

Reply:
even better to find a way to remove the global var entirely

Reply:
i was thinking about that yesterday for a moment, cause i agree i don't like the global vars stuff. I'll take another look.